packages/components/src/text-control/stories/index.tsx

```diff
@@ -19,7 +19,7 @@ const meta: ComponentMeta< typeof TextControl > = {
 	argTypes: {
 		help: { control: { type: 'text' } },
 		label: { control: { type: 'text' } },
-		type: { control: { type: 'text' } },
+		type: { control: { type: 'select' } },
```

Comment:
We can probably get rid of this line completely, since the type is simple enough for Storybook to understand it's a simple enum of values

Reply:
Makes sense 👍